plugin-offline/src/main/java/com/mapbox/mapboxsdk/plugins/offline/OfflineRegionSelector.java

```diff
@@ -15,6 +17,32 @@ private OfflineRegionSelector() {
     // No Instances
   }
 
+  public static DownloadOptions getDownloadOptions(final Intent data, byte[] metadata) {
```

Comment:
Add a ticket to this repo documenting that explanatory javadocs should be written for this class and its methods?

Reply:
https://github.com/mapbox/mapbox-plugins-android/issues/385 addresses this.